test/extended/util/annotate/rules.go

```diff
@@ -74,6 +74,17 @@ var (
 		"[Skipped:ovirt]": {},
 		"[Skipped:gce]":   {},
 
+		"[Skipped:SingleReplicaTopology]": {
+			`\[sig-apps\] Daemon set \[Serial\] should rollback without unnecessary restarts`,
```

Comment:
@rphillips @soltysh

Reply:
Looks like a good list.

Reply:
This a conformance test. Can you add the tag [Conformance] so that we can easily identify that we are skipping conformance tests as well, just like how you did for other conformance tests.

Reply:
you're right
done